2021-12-28 14:50:44,334 INFO: Initialized curve data.
2021-12-28 14:50:44,337 INFO: Setting up initial twist data...
2021-12-28 14:50:45,935 INFO: Twist setup complete.
2021-12-28 14:50:45,935 INFO: Searching for rational points on each twist...
2021-12-28 14:50:47,962 INFO: Finished searching for rational points on each twist.
2021-12-28 14:50:47,962 INFO: Testing local solvability of twists...
2021-12-28 14:51:08,399 INFO: Finished local solvability testing.
2021-12-28 14:51:08,401 INFO: Computing elliptic curve a-invariants...
2021-12-28 14:51:12,762 INFO: Finished computation of a-invariants.
2021-12-28 14:51:12,764 INFO: Computing MW group (delta = ['-16', '-52', '-25', '3', '-8', '-4'], g = [1, 3, 1])...
2021-12-28 14:51:15,038 INFO: Finished MW computation (delta = ['-16', '-52', '-25', '3', '-8', '-4'], g = [1, 3, 1])
2021-12-28 14:51:15,041 INFO: Computing MW group (delta = ['-16', '-52', '-25', '3', '-8', '-4'], g = [4, 1, -1, 1])...
2021-12-28 14:51:22,248 INFO: Finished MW computation (delta = ['-16', '-52', '-25', '3', '-8', '-4'], g = [4, 1, -1, 1])
2021-12-28 14:51:22,251 INFO: Skipped MW computation because no base point (delta = ['-4', '-8', '7', '6', '-2', '1'])
2021-12-28 14:51:22,251 INFO: Computing MW group (delta = ['-4', '-12', '-5', '1', '-2', '-1'], g = [1, 3, 1])...
2021-12-28 14:51:23,973 INFO: Finished MW computation (delta = ['-4', '-12', '-5', '1', '-2', '-1'], g = [1, 3, 1])
2021-12-28 14:51:23,976 INFO: Computing MW group (delta = ['-4', '-12', '-5', '1', '-2', '-1'], g = [4, 1, -1, 1])...
2021-12-28 14:51:26,553 INFO: Finished MW computation (delta = ['-4', '-12', '-5', '1', '-2', '-1'], g = [4, 1, -1, 1])
2021-12-28 14:51:26,556 INFO: Skipped MW computation because no base point (delta = ['4', '12', '7', '-1', '1', '2'])
2021-12-28 14:51:26,557 INFO: Computing MW group (delta = ['4', '12', '6', '-1', '2', '1'], g = [1, 3, 1])...
2021-12-28 14:51:29,586 INFO: Finished MW computation (delta = ['4', '12', '6', '-1', '2', '1'], g = [1, 3, 1])
2021-12-28 14:51:29,588 INFO: Computing MW group (delta = ['4', '12', '6', '-1', '2', '1'], g = [4, 1, -1, 1])...
2021-12-28 14:51:33,506 INFO: Finished MW computation (delta = ['4', '12', '6', '-1', '2', '1'], g = [4, 1, -1, 1])
2021-12-28 14:51:33,509 INFO: Computing MW group (delta = ['1'], g = [1, 3, 1])...
2021-12-28 14:51:36,556 INFO: Finished MW computation (delta = ['1'], g = [1, 3, 1])
2021-12-28 14:51:36,558 INFO: Computing MW group (delta = ['1'], g = [4, 1, -1, 1])...
2021-12-28 14:51:41,373 INFO: Finished MW computation (delta = ['1'], g = [4, 1, -1, 1])
2021-12-28 14:51:41,376 INFO: Skipped MW computation because no base point (delta = ['1', '-1', '0', '1', '-1'])
2021-12-28 14:51:41,377 INFO: Computing MW group (delta = ['-16', '-44', '-3', '3', '-15'], g = [1, 3, 1])...
2021-12-28 14:51:43,678 INFO: Finished MW computation (delta = ['-16', '-44', '-3', '3', '-15'], g = [1, 3, 1])
2021-12-28 14:51:43,682 INFO: Computing MW group (delta = ['-16', '-44', '-3', '3', '-15'], g = [4, 1, -1, 1])...
2021-12-28 14:51:53,103 INFO: Finished MW computation (delta = ['-16', '-44', '-3', '3', '-15'], g = [4, 1, -1, 1])
2021-12-28 14:51:53,110 INFO: Reducing MW generators...
2021-12-28 14:51:53,633 ERROR: An exception occurred. Recording data and exiting.
Traceback (most recent call last):
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 537, in <module>
    reduced_gens = reduce_MW_gens(gens_mag)
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 316, in reduce_MW_gens
    P += T[i][j] * indep_gens[j]
  File "sage/matrix/matrix0.pyx", line 1035, in sage.matrix.matrix0.Matrix.__getitem__ (build/cythonized/sage/matrix/matrix0.c:8317)
    r = self.row(row)
  File "sage/matrix/matrix1.pyx", line 1255, in sage.matrix.matrix1.Matrix.row (build/cythonized/sage/matrix/matrix1.c:14161)
    V = sage.modules.free_module.FreeModule(self._base_ring,
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 474, in FreeModule
    return FreeModuleFactory_with_standard_basis(base_ring, rank, sparse,
  File "sage/structure/factory.pyx", line 369, in sage.structure.factory.UniqueFactory.__call__ (build/cythonized/sage/structure/factory.c:2258)
    return self.get_object(version, key, kwds)
  File "sage/structure/factory.pyx", line 412, in sage.structure.factory.UniqueFactory.get_object (build/cythonized/sage/structure/factory.c:2652)
    obj = self.create_object(version, key, **extra_args)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 254, in create_object
    return FreeModule_ambient_pid(base_ring, rank, sparse=sparse)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 5522, in __init__
    FreeModule_ambient_domain.__init__(self, base_ring=base_ring,
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 5331, in __init__
    FreeModule_ambient.__init__(self, base_ring,
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 4694, in __init__
    FreeModule_generic.__init__(self, base_ring, rank=rank,
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 828, in __init__
    self.Element = element_class(coordinate_ring, sparse)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/free_module.py", line 7349, in element_class
    import sage.modules.vector_real_double_dense
ImportError: /share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/modules/vector_real_double_dense.cpython-39-x86_64-linux-gnu.so: failed to map segment from shared object: Cannot allocate memory
